Lib/test/test_uuid.py

```diff
@@ -21,7 +21,7 @@ def importable(name):
     try:
         __import__(name)
         return True
-    except:
+    except ImportError:
```

Comment:
```suggestion
    except ModuleNotFoundError:
```
Potentially?

Reply:
Yeah I think `ModuleNotFoundError` is better as it's even more specific